apollo-router/src/graphql/mod.rs

```diff
@@ -52,7 +54,7 @@ pub struct Location {
 /// as may be found in the `errors` field of a GraphQL [`Response`].
 ///
 /// Converted to (or from) JSON with serde.
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Default)]
```

Comment:
Removing Default is a public API change. We should not do this.

Reply:
Let me know if you are happy with the way I changed this.